pkg/scheduler/plugins/topology/job_filtering.go

```diff
@@ -39,50 +35,40 @@ type jobAllocationMetaData struct {
 	tasksToAllocate    []*pod_info.PodInfo
 }
 
-func (t *topologyPlugin) prePredicateFn(_ *pod_info.PodInfo, job *podgroup_info.PodGroupInfo) error {
-	topologyTree, err := t.getJobTopology(job)
+func (t *topologyPlugin) subSetNodesFn(podGroup *podgroup_info.PodGroupInfo, tasks []*pod_info.PodInfo, nodeSet node_info.NodeSet) ([]node_info.NodeSet, error) {
+	topologyTree, err := t.getJobTopology(podGroup)
```

Comment:
```suggestion
	topologyTree, err := t.getJobTopology(job)
```

I don't really like this name but usually it is a job